pulumi/rust_integration_tests/Pulumi.yaml

```diff
@@ -0,0 +1,3 @@
+name: integration-tests
```

Comment:
Probably need to rename this to `rust-integration-tests`, and make a similar change for the Python ones.